libtonode-tests/tests/sync.rs

```diff
@@ -75,10 +75,10 @@ async fn sync_test() {
     // .await
     // .unwrap();
 
-    increase_server_height(&regtest_manager, 1).await;
-    recipient.do_sync(false).await.unwrap();
-    recipient.quick_shield().await.unwrap();
-    increase_server_height(&regtest_manager, 1).await;
+    // increase_server_height(&regtest_manager, 1).await;
```

Comment:
Why is the shield test getting commented out? Is this restored by the verification fix followup pr?

Reply:
this test is just a template test i use to verify things as im developping the PoC. its not a fixed test that will stay in the codebase. integration tests are on the todo lists after PoC is complete